service/src/main/java/bio/terra/workspace/service/spendprofile/SpendProfileService.java

```diff
@@ -224,7 +224,7 @@ public SpendProfile createGcpSpendProfile(
   @VisibleForTesting
   public void deleteProfile(UUID profileId, AuthenticatedUserRequest userRequest) {
     try {
-      getProfileApi(userRequest).deleteProfile(profileId);
+      getProfileApi(userRequest).deleteProfile(profileId, userRequest.getSubjectId());
```

Comment:
this change required by the BPM client upgrade (should this send the subject id or an email?)